articles/azure/azs-how-create-key-vault-portal.md

```diff
@@ -0,0 +1,106 @@
+---
+title: How to create a Key Vault - Portal | UKCloud Ltd

```

Comment:
How to create a Key Vault using the Azure Stack portal

Reply:
Fixed